projects/Aerogramme/default.nix

```diff
@@ -1,10 +1,15 @@
-{ pkgs, ... }:
 {
-  packages = { inherit (pkgs) aerogramme; };
+  lib,
+  pkgs,
+  sources,
+}@args:
+{
+  name = "Aerogramme";
   nixos = {
     # https://aerogramme.deuxfleurs.fr/documentation/cookbook/config/
     # https://aerogramme.deuxfleurs.fr/documentation/cookbook/service-manager/
     modules.services = null;
+    modules.packages = { inherit (pkgs) aerogramme; };
```

Comment:
This needs to be a proper `programs` module, but not sure if Aerogramme even has standalone executables, so likely this line just needs to go away (which would correctly reflect taht we're not particularly far with packaging that one)

```suggestion
```

Reply:
Does that mean that we should keep `modules.packages`? If not, where do we put the packages in the new structure for the current overview?

Reply:
As discussed in one of the many discussions so far, the Nixpkgs notion of packages (which is largely a derivation with a bunch of data tacked on) is not particularly useful for us apart from providing files to work with, and it's merely an implementation detail for the our collection of NGI applications.